cases/common.js

```diff
@@ -375,7 +381,60 @@ function enablePerformanceTracking(useWebGL) {
   showGraph();
 }
 
+function animate() {
+  const view = map.getView();
+  const initialRotation = view.getRotation();
+  const initialZoom = view.getZoom();
+  const initialCenter = view.getCenter();
```

Comment:
I think it would give more consistent results if the initial parameters were hardcoded. Then before starting the animation chain the view parameters should be reset.

Reply:
@jahow  The initial parameters for rotation, zoom, and centre are now hardcoded.
Additionally, I have added a delay of one second after resetting the view parameters and before starting the animation chain.